adafruit_requests.py

```diff
@@ -442,14 +443,24 @@ def __init__(
         # Hang onto open sockets so that we can reuse them.
         self._open_sockets = {}
         self._socket_free = {}
-        self._last_response = None
 
-    def _free_socket(self, socket: SocketType) -> None:
+    def _free_sockets(self) -> None:
```

Comment:
This method stayed the same, just moved up as it's now the only pseudo private one